common/djangoapps/student/views.py

```diff
@@ -504,6 +504,7 @@ def complete_course_mode_info(course_id, enrollment, modes=None):
     if CourseMode.VERIFIED in modes and enrollment.mode in CourseMode.UPSELL_TO_VERIFIED_MODES:
         mode_info['show_upsell'] = True
         mode_info['verified_sku'] = modes['verified'].sku
+        mode_info['verified_bulk_sku'] = modes['verified'].bulk_sku
```

Comment:
Yeah, it's not anywhere in this file, from what I can see.  I've looked around for references to sku and I think I've got the pertinent ones